src/Ds2013/Presenters/Domain/CoreEntity/Programme/SubPresenters/ProgrammeBodyPresenter.php

```diff
@@ -27,6 +28,7 @@ class ProgrammeBodyPresenter extends ProgrammePresenterBase
         'body_suffix' => null,
         'show_child_availability' => false,
         'show_release_date' => false,
+        'show_masterbrand' => false
```

Comment:
Need comma on this line or linter will fail.
```suggestion
        'show_masterbrand' => false,
```